buddyboss/buddyboss-platform
# BP Theme settings

### All Submissions:


### Changes proposed in this Pull Request:

Check BB Theme illegal or not. If illegal then trigger flags.

### How to test the changes in this Pull Request:

1. Go to wp admin
2. See if notice appears.
